mattermost/mattermost
# [MM-56010]  Convert ./components/widgets/icons/accordion_toggle_icon.tsx from Class Component to Function Component

#### Summary
Converted the accordion_toggle_icon.tsx to functional component.
QA Steps: 

-    Navigate to System Console. 
-    Under USER MANAGEMENT -> Permissions -> click Edit Scheme
-    Check if the following button is working correctly.
![image](https://github.com/mattermost/mattermost/assets/58304531/311b6174-5384-4ce1-803a-47092a1b5538)




#### Ticket Link
https://github.com/mattermost/mattermost/issues/25602
https://mattermost.atlassian.net/browse/MM-56010


#### Screenshots: NA


#### Release Note

```release-note
NONE
```
